src/main/java/seedu/toluist/ui/MainWindow.java

```diff
@@ -136,11 +129,17 @@ private void setWindowMinSize() {
         primaryStage.setMinWidth(MIN_WIDTH);
     }
 
-//    @FXML
-//    public void handleHelp() {
-//        HelpWindow helpWindow = new HelpWindow();
-//        helpWindow.show();
-//    }
+    private void configureChildrenViews() {
+        taskListUiView = new TaskListUiView();
+        taskListUiView.setParent(getTaskListPlaceholder());
+
+        commandBox = new CommandBox(dispatcher);
+        commandBox.setParent(getCommandBoxPlaceholder());
+        commandBox.render();
```

Comment:
Is something wrong with this? 
configureChildrenViews shouldnt concern itself with render right?